test/e2e/apps/deployment.go

```diff
@@ -1476,3 +1480,75 @@ func waitForDeploymentCompleteAndCheckRolling(c clientset.Interface, d *appsv1.D
 func waitForDeploymentUpdatedReplicasGTE(c clientset.Interface, ns, deploymentName string, minUpdatedReplicas int32, desiredGeneration int64) error {
 	return testutil.WaitForDeploymentUpdatedReplicasGTE(c, ns, deploymentName, minUpdatedReplicas, desiredGeneration, poll, pollLongTimeout)
 }
+
+//Deployment should have a working scale subresource
```

Comment:
Comments should be separated with a space:
```suggestion
// Deployment should have a working scale subresource
```

Reply:
Updated